sktime/forecasting/base/adapters/_tbats.py

```diff
@@ -102,10 +102,39 @@ def _fit(self, y, X=None, fh=None):
         self._yname = y.name
 
         return self
+    
+    def _update(self, y, X=None, update_params=True):
+        """Update time series to incremental training data.
 
-    def _predict(self, fh, X):
-        """Forecast time series at future horizon.
+        Derived from example script provided by core devs in TBATS repository
+        https://github.com/intive-DataScience/tbats/blob/master/examples/re_fit_model.py
+
+        Parameters
+        ----------
+        y : pd.Series
+            Target time series to which to fit the forecaster.
+        X : pd.DataFrame, optional (default=None)
+            Exogenous variables (ignored)
+        update_params : bool, optional (default=True)
+            whether model parameters should be updated
 
+        Returns
+        -------
+        self : reference to self
+        """
+        if update_params:
```

Comment:
I think instead this should be as follows:

`if update_params`, then run `self._forecaster.fit`.

Else, *only return self*.

Why: `update_params=False` means data is ingested, but the internal models are *unchanged*.

Reply:
I think this is what happens by default (update model state only, not params) with the TBATS fit method if the model is not re-instantiated. 

A bit weird I know, so maybe I am misinterpretting. See example script here: https://github.com/intive-DataScience/tbats/blob/master/examples/re_fit_model.py. I was able to reproduce something similar for my own project.

Reply:
When ``update_params=False`` my understanding is that we want to change two things only: (a) self._y, (b) cutoff index within TBATS model class. We handle (a) I think already in the ``BaseForecaster`` class ``update`` method by calling ``_update_y_X`` but I think need to call TBAT's fit method for this to fix the cuttoff such that we can access other TBATS methods like forecast, etc. after the data update.

Reply:
I see, thanks for explaining!

Then this is ok.